README.md

```diff
@@ -20,12 +20,12 @@ Add an npm script to your package.json file to invoke `xata-codegen`. You can cu
 ```json
 {
   "scripts": {
-    "xata-codegen": "xata-codegen -s xata/schema.json -o xata.ts"
+    "xata-codegen": "xata-codegen xata -o generated/client.ts"
   }
 }
 ```
 
-Run it (this is assuming your schema file is in `xata/schema.json` and it was created by the `xata` cli application)
+Run it (this is assuming your Xata folder is in `./xata` and it was created by the `xata` cli application)
```

Comment:
The `folder` term is more common on UI apps, whereas the `directory` term is more used in command line apps. WDYT about using `directory` instead of `folder`

Reply:
Yeah, but I mean, not only here but in all references to `folder`.

Reply:
Sure, one sec...

Reply:
Done in bite-sized pieces in https://github.com/xataio/client-ts/pull/38/commits/d242e2990c16bf66b2dc19198c814be94d11a21b and https://github.com/xataio/client-ts/pull/38/commits/fa80ed34370f83677a63e26784b5f6540ed1c7a9. 😄